jenkins/databricks/test.sh

```diff
@@ -36,11 +61,34 @@ sudo "$(which pip)" install pytest sre_yield requests pandas pyarrow findspark p
 export SPARK_HOME=/databricks/spark
 # change to not point at databricks confs so we don't conflict with their settings
 export SPARK_CONF_DIR=$PWD
-export PYTHONPATH=$SPARK_HOME/python:$SPARK_HOME/python/pyspark/:$SPARK_HOME/python/lib/py4j-0.10.9-src.zip
-if [[ $BASE_SPARK_VERSION == "3.2.1" ]]
+# Get Python version from environment after lowercase and remove minor.
```

Comment:
The third version component is called micro https://docs.python.org/3/library/sys.html#sys.version_info